src/components/LightSwitch.js

```diff
@@ -1,11 +1,31 @@
-import React from "react"
+import React, { useState } from "react"
+//The `useState` hook will create a piece of state called `changeText` that represents the current state of the light switch.
 
+//Create a function called LightSwitch
 const LightSwitch = () => {
+
+//when the box is clicked ("onClick"), the function handleChange is called, so after clicking, the opposite of the current box state is returned
+
+  let [changeText, setChangeText] = useState(true);
+  const handleChange = () => {
+    return setChangeText(!changeText);
```

Comment:
Since we don't need to return anything the user will actually see, you don't have to put the "return" in this function. You can also have multiple things happen in the same function. So think about how you might be able to change the background color of the box at the same time you change the text within the same function.